client/lib/i18n-utils/utils.js

```diff
@@ -51,15 +51,13 @@ export function isLocaleVariant( locale ) {
 
 /**
  * Checks against a list of locales that don't have any GP translation sets
- *
+ * A 'translation set' refers to a collection of strings to be translated see:
+ * https://glotpress.blog/the-manual/translation-sets/
  * @param {string} locale - locale slug (eg: 'fr')
  * @return {boolean} true when the locale is a member of the exception list
  */
-export function hasTranslationSet( locale ) {
-	if ( ! isString( locale ) ) {
-		return false;
-	}
-	return [ 'en', 'sr_latin' ].indexOf( locale ) === -1;
+export function hasNoGlotPressTranslationSet( locale ) {
```

Comment:
I have several thoughts and questions on this function. 

First of all, I personally find this kind of functions very hard to follow, since it will result in the so-called double negation one day. e.g.
```
if ( someOtherCondition && ! hasNoGlotPressTranslationSet( 'aaa' ) )
```
It twists my brain a bit to understand it's about "someOtherCondition is true and aaa has a translation set. While I know some coding standards even explicitly being against this, I don't think we are enforcing this. Thus, this is more of a soft comment than a change request :)

Second, would state `GlotPress` here be too specific? From my view, Calypso actually has zero knowledge where the translation sets are from. It could be okay to keep it in the comment for clarifying the concept, but maybe not the function signature. Thus, I personally suggest that we remove `GlotPress` here. 

Third, it makes me wondering why we would need to hard code `[ 'en', 'sr_latin' ]` here. Since the locale variant is in-place now, is there still any good reason `sr_latin` doesn't has a translation set? If we could eliminate it, this function could be simply checking the passed in `locale` argument is not the default locale.

Reply:
> First of all, I personally find this kind of functions very hard to follow, since it will result in the so-called double negation one day. e.g.

Thanks for raising this @southp. What you say makes sense. The whole exception logic is a bit of a contortion. I chose ease of implementation as priority.

What about something like `shouldEnableCommunityTranslatorForLocale( locale )` or `canBeTranslated( locale )` as a method name? 

It's specific to the Community Translator, and might better explains its function. 

> Second, would state GlotPress here be too specific? From my view, Calypso actually has zero knowledge where the translation sets are from.

This is a fair point. Agree. See above.

> Since the locale variant is in-place now, is there still any good reason sr_latin doesn't has a translation set?

`en` had always been hardcoded, however, as for `sr_latin` I don't know why there is no GlotPress translation set. @yoavf ?

Reply:
> What about something like shouldEnableCommunityTranslatorForLocale( locale ) or canBeTranslated( locale ) as a method name?

Since we might rename CT in the near future, I think something like `canBeTranslated( locale )` is good and easier to read for me :)